### ISSUE TITLE
Should be possible to drag fillHandle with keyboard navigation

### ISSUE THREAD

**Issue by taauntik** [bryntum/support#8651]
[Forum post](https://forum.bryntum.com/viewtopic.php?p=141055#p141055)

One final request:
Is it possible to corner drag using keyboard navigation? If not, would it be possible to add this feature, too?

**Comment by marciogurka** [comment]
[+1](https://forum.bryntum.com/viewtopic.php?p=144451#p144451) for this
